services/user.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from db.models import User

```

Comment:
The import statement `from db.models import User` is unnecessary because the `User` model is not directly used in this file. You can remove this import to clean up the code.